CHANGELOG.md

```diff
@@ -7,11 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [v0.1.9] 2020-05-04
+## [v0.1.9] 2020-05-06
 ### Added
 - The operator passes the configured annotation names to Wrestic
 ### Fixed
 - KeepJobs value of sub items in Schedule are not overridden
+### Changed
```

Comment:
That should go into unreleased, as v0.1.9 is already released.

Reply:
Fixed. -- Sorry it took so long!

Reply:
Note that the latest released version of the docs is in fact [0.1.8](https://k8up.io/). Version 0.1.9 doesn't seem to have been published (automatically) with the software release.

Reply:
Yes, looks like the docs aren't build automatically. We should fix that.